znoyder/generator.py

```diff
@@ -170,6 +170,8 @@ def main(argv=None) -> None:
                                             directory))
         jobs = finder.find_jobs(path, templates, triggers)
 
+        # Case where zuul configs are inside zuul.d
+        directory = directory.split('/zuul.d')[0].split('.zuul.d')[0]
```

Comment:
For some reason I consider `.replace('/zuul.d', '').replace('/.zuul.d', '')` a little better here, what do you think?